command.go

```diff
@@ -404,3 +408,20 @@ func hasCommand(commands []*Command, command *Command) bool {
 
 	return false
 }
+
+func checkDuplicatedCmds(parent *Command) error {
+	if parent == nil {
+		return nil
+	}
```

Comment:
Why there is a need to call the function with `nil` argument?

Reply:
@abitrolly  It's a defensive way too. Already deleted.